src/pool.c

```diff
@@ -239,6 +239,7 @@ typedef struct payment_t
     uint64_t amount;
     time_t timestamp;
     char address[ADDRESS_MAX];
+    unsigned char error;
```

Comment:
Oh and I forgot to mention, this would require a DB migration on the payments DB as it changes the size of the structure.

Reply:
To be clear, if you want to keep this, you have to reduce the size of the address field. Otherwise it will break existing databases.